source/ModchartState.hx

```diff
@@ -312,6 +328,7 @@ class ModchartState
 				// get some fukin globals up in here bois
 	
 				setVar("difficulty", PlayState.storyDifficulty);
+				setVar("health", PlayState.instance.health);
```

Comment:
No point in having this since getProperty(); exists, soon with setProperty(); as well